dev-util/cmark/cmark-0.26.1.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="CommonMark parsing and rendering library and program in C"
+DESCRIPTION="cmark is the C reference implementation of CommonMark, \
+a rationalized version of Markdown syntax with a spec. \
+(For the JavaScript reference implementation, see commonmark.js.) \
+It provides a shared library (libcmark) with functions for parsing \
+CommonMark documents to an abstract syntax tree (AST), manipulating the AST, \
+and rendering the document to HTML, groff man, LaTeX, CommonMark, or an XML \
+representation of the AST. It also provides a command-line program (cmark) \
+for parsing and rendering CommonMark documents."
+HOMEPAGE="https://github.com/jgm/cmark"
+COPYRIGHT="2014 John MacFarlane"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/jgm/cmark/archive/0.26.1.tar.gz"
```

Comment:
You might wish to use:
```
SOURCE_URI="https://github.com/jgm/cmark/archive/$portVersion.tar.gz" 
```
This is very handy for software that have frequent updates ;-)